airbyte-workers/src/test/java/io/airbyte/workers/temporal/TemporalAttemptExecutionTest.java

```diff
@@ -68,19 +71,23 @@ void setup() throws IOException {
     mdcSetter = mock(BiConsumer.class);
     jobRootDirCreator = mock(CheckedConsumer.class);
 
-    attemptExecution = new TemporalAttemptExecution<>(workspaceRoot, JOB_RUN_CONFIG, execution, mdcSetter, jobRootDirCreator);
+    attemptExecution = new TemporalAttemptExecution<>(workspaceRoot, JOB_RUN_CONFIG, execution, () -> "", mdcSetter, jobRootDirCreator,
+        mock(CancellationHandler.class));
   }
 
   @Test
   void testGet() throws Exception {
```

Comment:
nit: I've been slowly introducing JUnit 5's `@DisplayName` annotation in the test classes I write. It's nice as it provides more human readable context in general. It's very helpful as test get more complicated. I don't necessary think it's needed here, since these are simple. Just letting ya know in case you haven't heard of it :)